app/brave_command_line_helper.cc

```diff
@@ -93,3 +93,10 @@ void BraveCommandLineHelper::AppendCSV(
   }
   command_line_.AppendSwitchASCII(switch_key, ss.str());
 }
+
+void BraveCommandLineHelper::AppendSwitchASCII(
```

Comment:
Needed for: https://github.com/brave/brave-core/pull/2471/files#diff-9413752841345e7c426bf4dd82fe7e60R134